openedx/core/lib/django_startup.py

```diff
@@ -1,11 +1,25 @@
 """
-Automatic execution of startup modules in Django apps.
+Extend startup behavior shared between LMS and CMS
 """
 
 from importlib import import_module
 from django.conf import settings
 
 
+def add_mimetypes():
```

Comment:
No. This code already exists upstream; I'm just putting it somewhere that both LMS and CMS can import.